### ISSUE TITLE
Investigate klog redirect

### ISSUE THREAD

**Issue by DirectXMan12** [kubernetes-sigs/controller-runtime#201]
It looks like klog has an option to redirect output to a writer.  We should investigate setting this up to redirect to logger so that the client-go logging doesn't go into the ether.

**Comment by DirectXMan12** [comment]
/remove-lifecycle rotten

still waiting on klog for this, IIRC

**Comment by DirectXMan12** [comment]
just merged into klog, but we'll need to wait till v2 branches due to compat issues
